packages/devtools_app/lib/src/screens/memory/panes/profile/profile_pane_controller.dart

```diff
@@ -15,6 +15,18 @@ import 'model.dart';
 
 class ProfilePaneController extends DisposableController
     with AutoDisposeControllerMixin {
+  ProfilePaneController();
+
+  factory ProfilePaneController.parse(Map<String, dynamic> map) {
```

Comment:
use Object? instead of dynamic. Here and elsewhere in this PR

Reply:
It is consistent with what is recommended for serialization: https://docs.flutter.dev/data-and-backend/serialization/json#serializing-json-inside-model-classes

Map<String, dynamic> is a standard for json.

Reply:
Commented above. Those docs look outdated.

Reply:
Commented above.

Reply:
If docs are outdated, how about first updating the doc and then starting following the updated guidance?